docs/asciidoc/static/breaking-changes.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[breaking-changes]]
+== Breaking changes
+
+Version 2.0 of Logstash has some changes that are incompatible with previous versions of Logstash. This section discusses 
+what you need to be aware of when migrating to this version.
+
+[float]
+=== Elasticsearch Output Default
+
+Starting with the 2.0 release of Logstash, the default Logstash output for Elasticsearch is HTTP. To use the `node` or
+`transport` protocols, download the {logstash}plugins-outputs-elasticsearch_java.html[Elasticsearch Java plugin]. The 
```

Comment:
Isn't it installed just like any other plugin? We've already got docs on
plugin management -- we can link to them from here.

On Wed, Oct 14, 2015 at 7:34 PM, Suyog Rao <notifications@github.com> wrote:

> In docs/asciidoc/static/breaking-changes.asciidoc
> <https://github.com/elastic/logstash/pull/4039#discussion_r42077893>:
>
> > @@ -0,0 +1,30 @@
> > +[[breaking-changes]]
> > +== Breaking changes
> > +
> > +Version 2.0 of Logstash has some changes that are incompatible with previous versions of Logstash. This section discusses
> > +what you need to be aware of when migrating to this version.
> > +
> > +[float]
> > +=== Elasticsearch Output Default
> > +
> > +Starting with the 2.0 release of Logstash, the default Logstash output for Elasticsearch is HTTP. To use the `node` or
> > +`transport` protocols, download the {logstash}plugins-outputs-elasticsearch_java.html[Elasticsearch Java plugin]. The
>
> can we add instructions on how to install it? See details here:
> https://www.elastic.co/blog/logstash-2-0-0-beta1-released "Installing
> java clients for Elasticsearch 1.x:"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/logstash/pull/4039/files#r42077893>.
>